### DISCUSSION TITLE
Re-run PR checks when Actions commits & pushes to the branch

### DISCUSSION THREAD

**Discussion by iansan5653** [community/community#26269] [Actions]
<p>Hello, I have a workflow that runs on PRs and formats code before pushing it to the branch that the PR is from. This works (it appears on the pull request), but the workflows don’t re-run afterwards, so I can’t be sure that the workflow didn’t break anything. How can I get them to re-run automatically? Here you can see what it looks like, and how the actions don’t appear at all after the push:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/611af03b5d6b1e4c4c72681a53ac970f404d365b.png" data-download-href="/uploads/short-url/dR21QWsh4Vy4qrSGxnvlo61TksH.png?dl=1" title="Untitled.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/2294248/181096569-a7ec096f-612b-49e6-898d-8841adeb0bff.png" alt="Untitled.png" data-base62-sha1="dR21QWsh4Vy4qrSGxnvlo61TksH" width="690" height="309" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/611af03b5d6b1e4c4c72681a53ac970f404d365b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Untitled.png</span><span class="informations">786×352 17.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by crash7** [comment, marked as answer]
<p>Hi,<br>
You need to use a different github token if you need to push code from an action. In order to prevent abuse, actions can trigger other actions (this is by design).</p>
<p>You will need to create a personal github token and store it as a secret in your repo.</p>

**Comment by suryasr007** [comment]
<p>I’ve tried it. It didn’t work. What should be the scope of the personal_access_token? Is there any example workflow?</p>

**Comment by crash7** [comment]
<p>The repo scope (push access basically) should work.</p>

**Comment by zalegrala** [comment]
<p>Is it not sufficient to have the appropriate permission on the action action?  I was under the assumption that the token supplied to the action would have the appropriate permissions.  The push is successful, but the jobs aren’t run.</p>
